assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}.`);
+    }
+}
+
+class Instructor extends Person {
+    constructor(instAttributes) {
+        super(instAttributes);
+        this.specialty = instAttributes.specialty;
+        this.favLanguage = instAttributes.favLanguage;
+        this.catchPhrase = instAttributes.catchPhrase;
+    }
+    demo(subject) {
+        console.log(`Today we are learning about ${subject}.`);
+    };
+    grade(student, subject) {
+        console.log(`${student.name} receives a perfect score on ${subject}.`);
+    };
+    calculateGrade(student) {
+        let rando = Math.floor(Math.random() * Math.floor(10));
+        if (rando %2 === 0) {
+            return student.grade = (student.grade - rando);
+        } else if (rando %2 !== 0) {
```

Comment:
Here, you can simply use `else` and forgo the longer `else if` conditional expression.